src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -308,7 +308,8 @@ protected function getCategoryListRepresentation(Request $request, $parentKey =
         $listBuilder->addSelectField($fieldDescriptors['locale']);
         $listBuilder->addSelectField($fieldDescriptors['defaultLocale']);
 
-        $listBuilder->addGroupBy($fieldDescriptors['id']);
+        // I think this "group by" is not required
+        //$listBuilder->addGroupBy($fieldDescriptors['id']);
```

Comment:
I had it to the CHANGELOG.md but travis continuous integration failed ;-)
I try again ;)